views/layout.html

```diff
@@ -25,6 +25,12 @@
       <script type="text/javascript" src="{{assetPath}}/js/bundle.js"></script>
     </main>
   {{/main}}
+  {{$cookieMessage}}
+    <p>
+      {{#t}}cookies.banner.start{{/t}}
```

Comment:
did you mean to add cookies.banner.begin as per the cookies.json file above https://github.com/UKHomeOfficeForms/hof-template-partials/pull/30/files#diff-cbed92cb1e8f1380315ac0d94f535b5cR3